setup.cfg

```diff
@@ -1,2 +1,5 @@
 [bdist_wheel]
 universal = True
+
+[metadata]
+license-file = LICENSE.txt
```

Comment:
what does this license-file does exactly? (could not find it in the distutils documentation yet)